### ISSUE TITLE
T58830 : Export GLTF/GLB - 2-nd time in 1 session won't work

### ISSUE THREAD

**Issue by julienduroure** [KhronosGroup/glTF-Blender-IO#148]
Issue from Blender tracker :

https://developer.blender.org/T58830

**Comment by emackey** [comment]
In the linked issue, the scene has these glTF export settings:

```
{'export_cameras': 1, 'export_format': 0, 'export_lights': 1, 'export_texture_transform': 1}
```

This is a duplicate of #146.

I would consider this very high priority now.  Here are steps to reproduce from scratch:

1. Launch Blender 2.80 Beta
2. File -> Export glTF, to bring up the export options panel.
3. In export settings, (a) change type to "glTF Embedded", and (b) click "Save Settings" pushpin at the bottom, so that the scene custom property will be created.
4. Choose a filename and complete the export of the default cube.
5. Go back to File -> Export glTF a second time.  It will crash.

**Comment by EJTH** [comment]
I had the same problem. Seems like this change works as intended:

https://github.com/EJTH/glTF-Blender-IO/commit/15cb252d08a2ba325a3f0e43e5f02298cf94c190#diff-42dbf07f348e019791ca29e531182c7c

**Comment by emackey** [comment]
This was fixed for "new" saved settings in #150.

Blend files with saved settings from the past few days will continue to crash the exporter, until the user goes into Scene Properties -> Custom Properties and deletes the `glTF2ExportSettings`.  For this reason, we might want to consider re-opening and merging #149.

**Comment by emackey** [comment]
@julienduroure This should be fixed now with your next push to the Blender repo.
